cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -1,11 +1,10 @@
-kind: StatefulSet
+kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: csi-cephfsplugin-provisioner
   namespace: {{ .Namespace }}
 spec:
-  serviceName: "csi-cephfsplugin-provisioner"
-  replicas: 1
+  replicas: 3
```

Comment:
@travisn yes it's for availability, this is copied from external-provisioner example.  I feel its better have 3 copies for HA. if it's required I can make it as 2.

Reply:
I still see that 2 is what we want. We don't need quorum (like mons or etcd), we just need one to respond. Three would be overkill since K8s is already going to be keeping each individual pod running.